index.html

```diff
@@ -1,114 +1,139 @@
-                <!--Changed the order of elements from img,heading,paragraph to h2,img,p elements-->
-            <h2>Social Media Marketing</h2>
-
-            <img src="./assets/images/social-media-marketing.jpg" alt ="Social Media Marketing" class="float-left" />
-            
-            <p>
-                Social media continues to have a sizable influence on buying habits. Social media marketing helps you determine which platforms are suited to your brand, using analytics to find the right markets and increase your lead generation.
-            </p>
-        </section>
-    </main>
-    <!--It seems the content is slightly related to the webapge,so changed div to aside-->
-    <aside class="benefits">
-        <!--Changed div to article tag,since it is an independent content-->
-        <article class="benefit-lead">
-            <h3>Lead Generation</h3>
-            <!--Added alt attribute for the images below,when problem loading image,alt text would shown in the webpage-->
-            <img src="./assets/images/lead-generation.png" alt="Lead Generation" />
-            <p>
-                Inbound strategies for lead generation require less work for your business, bringing customers directly to your website.
-            </p>
-        </article>
-        <article class="benefit-brand">
-            <h3>Brand Awareness</h3>
-            <img src="./assets/images/brand-awareness.png" alt ="Brand Awareness"/>
-            <p>
-                Users find your business through paid and organic searches, increasing the search ranking and visibility for your business.
-            </p>
-        </article>
-        <article class="benefit-cost">
-            <h3>Cost Management</h3>
-            <!--img is the self closing tag,so added the closing in tag itself instead of a seperate closing tag-->
-           
-            <img src="./assets/images/cost-management.png" alt="Cost Management"/>
-       
-            <p>
-                As the search ranking for your business increases, your advertising costs decrease, and you no longer need to advertise your page.
-            </p>
-        </article>
-    </aside>
-    <!--Since it is an footer, changing div to footer -->
-    <footer class="footer">
-        <!--since heart is a figure,so added figure element and html entity for the heart emoji-->
-
-        <h2>Made with &#10084;&#65039; by Horiseon</h2>
-        
-            <p>&copy; 2019 Horiseon Social Solution Services, Inc.</p>
-        
-    </footer>
+  <!--Since it is an header, changing div to header -->
+  <header class="header">
+
+    <!-- ~~~~~~~~~~ Dope! you got everything basically just some extramile things ~~~~~~~~~~ -->
+    <!-- **********
+      this portion right here can be altered to be read more clearly and two ways to go about it for me 
+    -->
+    <h1>Hori<span class="seo">seo</span>n</h1>
+    <!-- ~~~~~~~~~~ <h1 class="seo">Horiseon</h1> ~~~~~~~~~~-->
+    <!-- ~~~~~~~~~~ <h1>Horiseon</h1> ~~~~~~~~~~ -->
```

Comment:
@daniel-thao ,Thanks for the feedback , Initially I did what you commented, then I thought their logo has two colors , and so I reverted it.Please find my initial commit https://github.com/Anitha-Venkatesan/CodeRefactor-Assignment1/commit/f225380917c422cff422a06d887a562b8a6057d9